pkg/probe/probe.go

```diff
@@ -45,7 +50,12 @@ func (p *probeValue) DeepCopyMapValue() bpf.MapValue { return &probeValue{p.Valu
 // HaveFullLPM tests whether kernel supports fully functioning BPF LPM map
 // with proper bpf.GetNextKey() traversal. Needs 4.16 or higher.
 func HaveFullLPM() bool {
+	if checkedFullLPM {
+		return haveFullLPM
```

Comment:
nit: As a matter of style/future-proofing, we would typically use a `sync.Once()` to ensure no future concurrency issues with a function like this.